scripts/shared/content-import.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+STARTERKIT_VERSION="latest"
+STARTERKIT_FILE="starterkit.zip"
+DOWNLOAD_URL="https://github.com/yalesites-org/yalesites-starterkit/releases/$STARTERKIT_VERSION/download/$STARTERKIT_FILE"
+
+# Download starterkit export.
+if curl -s -O -L "$DOWNLOAD_URL"; then
+  echo "Starterkit file downloaded successfully."
+else
+  echo "Failed to download starterkit file."
+  exit 1
+fi
+
+# Check if running under lando, otherwise assume CI.
+# Put starterkit file in place and import.
+# Clean up, and set front page in config.
+if [ "$YALESITES_IS_LOCAL" == "1" ]; then
+  lando drush content:import ../"$STARTERKIT_FILE"
+  lando drush cset system.site page.front '/homepage' -y
```

Comment:
We had talked a little bit about this one to set this as `/node/xx` instead of `/homepage`. We could either run some PHP code on the CLI via `php eval` or perhaps write a custom drush command to parse a path like `/homepage` and turn that into the internal path needed `/node/xx` to then use for the config we need to set here. I'm happy to look into creating a drush command if you'd like as I've never done that before and would be kind of neat to see how to get that to work.

Reply:
Is Linkit the problem here? A clean D10 install of the standard profile allows us to use an alias without a problem, so I think we should make this work.

![Screenshot 2023-12-11 at 4 32 52 PM](https://github.com/yalesites-org/yalesites-project/assets/969062/6280c288-b1a6-4b91-b5d5-ee84923f0492)

Reply:
OK, I just tested Linkit and it correctly does choose the `/node/xx` when using autocomplete. The core Drupal site settings form (which we aren't using) will convert it to a `/node/xx` in config even though in the form, it shows the alias.

I think the Linkit change is new, as IIRC, it didn't do that before - it saved the alias before. So, the main issue is for any new sites and if we set it to `/homepage` in config via the script, it won't get the `/node/xx`. You have to go into the Linkit form to set it correctly.

I put this in Teams, but I just created a drush command (as part of YS Core) to get the internal path when you specify a path alias, so we could use that to get the correct internal path and set that in the script dynamically. `drush intpath /alias-to-page`

Reply:
@codechefmarc I found an active issue in the Linkit queue for this: https://www.drupal.org/project/linkit/issues/2877535

I tested patch # 64 locally and it seems to work fine setting `/homepage`. While the Drush command would work, I think it isn't needed if we use the patch, and this should eventually get fixed in that issue.

Reply:
So, I retested with the change of Linkit, and it still doesn't work for me when doing a clean site install using the starter kit scripts. This is because we're setting the config value of `system.site` `page.front` to a string `/homepage` and not specifically the internal node id which it needs to not have the `/homepage` in the URL when visiting the front page. So, we should be setting it to `/node/xx` like Drupal does internally.

Also, this change with Linkit causes a reversion where it, too, is saving the config as a string:

```
page:
  403: ''
  404: ''
  front: /homepage
```

Whereas if I set it via Drupal's form:

```
page:
  403: ''
  404: ''
  front: /node/4
```

So, I think we do need to either use the Drush command or find someway to save the homepage config when exporting content from the starter kit site?

Reply:
Oh, this affects not only the URL having the extra path on the front page, but also, with the new focus header, there is a core function to check to see if you are on the homepage and that only works if the config is set to `/node/xx` and won't work if it's set to an alias. It's annoying, hah.